CONTRIBUTING.md

```diff
@@ -72,15 +83,14 @@ yum install gcc gcc-c++ make
 Windows:
 
 ```
-npm install -g windows-build-tools
+pnpm add -g windows-build-tools
```

Comment:
Ideally if @maspio can test on Win.

Reply:
I tested this on bash on windows (no WSL), as well as on powershell.